lib/improver/cli/extend_radar_mask.py

```diff
@@ -35,17 +35,17 @@
 
 @cli.clizefy
 @cli.with_output
-def process(coverage: cli.inputcube,
-            radar_data: cli.inputcube):
+def process(cube: cli.inputcube,
```

Comment:
What's the reason to use `cube` as the variable name, in this CLI and elsewhere? `cube` seems like quite a vague name for a CLI argument.

Reply:
Consistency.  Where a step is essentially modifying an input field (eg extending the mask, neighbourhood processing a probability) the input name is `cube`.  Where a new diagnostic or ancillary is being created (eg lapse rate, topography band ancillaries), the inputs are named diagnostics.

This is an odd case as we do expect it always to be radar data.  However I don't disagree with the idea of having any "cube to be modified" consistently named as `cube` at the CLI level.

Reply:
Just having `input` would look good on the command line. (But not necessarily nice for a python function)

Reply:
Yes, this was exactly my reasoning. I've also considered `input` and `input_cube`, but `cube` was by far the most used one, so I decided to stick with it. If any of you have a strong preference for another name let me know asap.

BTW, I think the original order of arguments to this CLI was wrong (based on plugin docs), but could someone else please verify that?